wpxmlrpc.podspec

```diff
@@ -1,16 +1,15 @@
 Pod::Spec.new do |s|
   s.name         = "wpxmlrpc"
-  s.version      = "0.8"
+  s.version      = "0.8.2"
   s.summary      = "Lightweight XML-RPC library."
   s.homepage     = "https://github.com/wordpress-mobile/wpxmlrpc"
-  s.license      = { :type => 'MIT', :file => 'LICENSE.md' }
-  s.author       = "WordPress"
+  s.license      = { :type => 'GPLv2', :file => 'LICENSE.md' }
+  s.author       = {"WordPress", "Automattic" => "mobile@automattic.com"}
   s.source       = { :git => "https://github.com/wordpress-mobile/wpxmlrpc.git", :tag => "#{s.version}" }
   s.source_files = 'WPXMLRPC'
-  s.public_header_files = [ 'WPXMLRPC/WPXMLRPC.h', 'WPXMLRPC/WPXMLRPCEncoder.h', 'WPXMLRPC/WPXMLRPCDecoder.h' ]
```

Comment:
Because I changed the structure of the test project to use development pod instead of directly including the files, I need access to all h file in order to make the tests compile.